packages/core/primitives/event-dispatch/src/earlyeventcontract.ts

```diff
@@ -52,7 +52,7 @@ export class EarlyEventContract {
           event.type,
           event,
           event.target as Element,
-          window.document.documentElement,
+          this.container,
```

Comment:
nit: can we refer to the `container` without `this` (to save some bytes)?
```suggestion
          container,
```